yarn.lock

```diff
@@ -353,7 +353,7 @@
     "@babel/traverse" "^7.6.2"
     "@babel/types" "^7.6.0"
 
-"@babel/highlight@^7.10.4":
+"@babel/highlight@^7.10.4", "@babel/highlight@^7.8.3":
```

Comment:
not sure why yarn changed that 🤔  anyways, it should not cause any issues

Reply:
@Malinicz 

it means that one of dependency was found to be using: "@babel/highlight@^7.8.3" but is satisfied by the the same: version "7.10.4". This is how yarn is trying to merge different lock version in a defensive way.

This is correct since eg. @babel/code-frame@7.8.3 and @babel/code-frame@^7.0.0 are present and are using different version. yarn lock will grow over time and will have such a duplicates unless is cleaned.

Reply:
cool, good to know 👍